pass-bakery/src/features/redux/cart.js

```diff
@@ -0,0 +1,98 @@
+import { useDispatch, useSelector } from "react-redux";
+
+import React from "react";
+import { remove } from "./cartSlice";
+import styled from "styled-components";
+
+export default function Cart(props) {
+  const cart = useSelector((state) => state.cart);
+  const dispatch = useDispatch();
+  const itemQuantity = cart.items.length;
+  let price = 0;
+
+  for (let i = 0; i < itemQuantity; i++) {
+    price += cart.items[i][1];
+  }
```

Comment:
les move the price calculation done on the reducer, should happen when `add` or `remove` gets called.